ChopChop/Models/Recipes/Recipe.swift

```diff
@@ -10,9 +10,10 @@ class Recipe: FetchableRecord, ObservableObject {
     @Published private(set) var difficulty: Difficulty?
     @Published private(set) var steps: [RecipeStep]
```

Comment:
Yes they should be removed. I didn't remove them because it would cause many front end stuff to not compile. I think you'll probably have an easier time integrating the change because you've already modified said front end stuff